README.md

```diff
@@ -90,4 +79,4 @@
 - [ ] **[Footer section]**
 - [ ] **[form section]**
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Good job for documenting your README File Professional, But I recommend to include "Getting Started" section and the section and All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in a your's repo.

Reply:
- [ ] Kindly implement the changes required by the previous reviewer.